code/Maximum sum increasing.cpp subsequence

```diff
@@ -0,0 +1,47 @@
+/* Dynamic Programming implementation
+of Maximum Sum Increasing Subsequence
+(MSIS) problem */
+#include <bits/stdc++.h>
+using namespace std;
+
+/* maxSumIS() returns the maximum
+sum of increasing subsequence
+in arr[] of size n */
+int maxSumIS(int arr[], int n)
+{
+	int i, j, max = 0;
+	int msis[n];
+
+	/* Initialize msis values
+	for all indexes */
+	for ( i = 0; i < n; i++ )
```

Comment:
hey can we use this line of code here 
```copy (arr, arr + n, msis);``` 
instead of  
```
for ( i = 0; i < n; i++ )
    msis[i] = arr[i];
```
Which reduce the lines of code?